packs/rabbitmq/templates/rabbit.nomad.tpl

```diff
@@ -0,0 +1,107 @@
+job [[ template "job_name" . ]] {
+
+  datacenters = [[ .rabbitmq.datacenters | toJson ]]
+  type        = "service"
+
+  constraint {
+    attribute = "${attr.consul.version}"
+    operator  = "is_set"
+  }
+
+  group "cluster" {
+    count = [[ .rabbitmq.cluster_size ]]
+
+    update {
+      max_parallel = 1
+    }
+
+    migrate {
+      max_parallel     = 1
+      health_check     = "checks"
+      min_healthy_time = "5s"
+      healthy_deadline = "30s"
+    }
+
+    network {
+      port "amqp" {
+        to = 5671
+        [[- template "port" .rabbitmq.port_amqp ]]
+      }
+      port "ui" {
+        to     = 15671
+        [[- template "port" .rabbitmq.port_ui ]]
+      }
+      port "discovery" {
+        to = 4369
+        static = 4369
+      }
+      port "clustering" {
+        to = 25672
+        static = 25672
+      }
+    }
+
+    task "rabbit" {
+      driver = "docker"
+
+      [[ if .rabbitmq.vault_enabled -]]
+      vault {
+        policies    = [[ .rabbitmq.vault_roles | toJson ]]
+        change_mode = "restart"
+      }
+      [[- end ]]
+
+      config {
+        image      = "[[ .rabbitmq.image ]]"
+        hostname   = "${attr.unique.hostname}"
+        ports      = ["amqp", "ui", "discovery", "clustering"]
+
+        volumes = [
```

Comment:
Should we call out the location of these volumes on the host in the README?

Reply:
I think so, although they are just the task local volumes.   It's a shame the `artifact` stanza can't write directly to paths inside the containers, but it makes sense that it can't.

Reply:
Added to the readme